polkadot/zombienet_tests/functional/0016-approval-voting-parallel.toml

```diff
@@ -0,0 +1,120 @@
+[settings]
+timeout = 1000
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config]
+  needed_approvals = 4
+  relay_vrf_modulo_samples = 6
```

Comment:
This means all assignments in this test will be tranche0. I think we should set this to 2 to ensure we see other tranches triggering.